services/brig/test/integration/API/User/Account.hs

```diff
@@ -1632,18 +1633,91 @@ testTooManyMembersForLegalhold opts brig = do
         const 403 === statusCode
         const (Right "too-many-members-for-legalhold") === fmap Wai.label . responseJsonEither
 
+testDeleteUserWithCompletelyDeletedUser :: Brig -> Cannon -> AWS.Env -> Http ()
+testDeleteUserWithCompletelyDeletedUser brig cannon aws = do
+  u <- randomUser brig
+  liftIO $ Util.assertUserJournalQueue "user activate testDeleteUserWithCompletelyDeletedUser" aws (userActivateJournaled u)
+  setHandleAndDeleteUser brig cannon u [] aws $
+    \uid -> deleteUserInternal uid brig !!! const 202 === statusCode
+  do
+    let uid = userId u
+    deleteUserInternal uid brig
+      !!! do
+        const 200 === statusCode
```

Comment:
```suggestion
  deleteUserInternal (userId u) brig !!! const 200 === statusCode
```

(and similarly below)

(question of taste, feel free to ignore)

Reply:
Hm, I think most of it was an unconscious decision (i.e. `ormulu`).